wps/controllers/urlServer.classic.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+class urlServerCtrl extends jController
+{
+    // Configure access via jacl2 rights management
+    public $pluginParams = array(
+        '*' => array('jacl2.right' => 'wps.urlserver.manage'),
+    );
+
+    private $iniFile;
+
+    private $wpsSection = 'wps';
+
+    public function __construct($request)
+    {
+        parent::__construct($request);
+        $file = jApp::varConfigPath('liveconfig.ini.php');
+        $this->iniFile = new \Jelix\IniFile\IniModifier($file);
+    }
+
+    /**
+     * Display a summary of the information taken from the ~ configuration file.
+     *
+     * @return jResponseHtml
+     */
+    public function show()
+    {
```

Comment:
call in all actions `\lizmap::getServices()->hideSensitiveProperties()` to check if it is forbid (so return a 403 error) or not.